index.html

```diff
@@ -0,0 +1,260 @@
+                       </a>
+                      
+                      </div>
+                  </div>
+                  <div class="col-md-4 col-lg-4">
+                    <span class="copy-right">
+                        © 2021 NEWSWEEK DIGITAL LLC
+                        </span>
+                  </div>
+                  <div class="col-md-4 col-lg-4 ">
+                      <div class="social-icons float-right">
+                        <a href="#" class="fa fa-facebook"></a>
+                        <a href="#" class="fa fa-twitter"></a>
+                        <a href="#" class="fa fa-instagram"></a>
+                        <a href="#" class="fa fa-tumblr"></a>
+                        <a href="#" class="fa fa-linkedin"></a>
+                      </div>
```

Comment:
- [ ] Your icons don't look great right now. I understand if you cant find the same icons as used in the clone but maybe try resizing some of them so it looks better.